File: server/order-service/src/main/java/com/sudo248/orderservice/controller/order/dto/UpsertOrderPromotionDto.java
package com.sudo248.orderservice.controller.order.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UpsertOrderPromotionDto {
    private String promotionId;
    private String orderSupplierId;
    private Double totalPrice, totalPromotionPrice, finalPrice;
}

File: client/mobile/Sudoo/app/src/main/java/com/sudo248/sudoo/data/dto/order/UpsertOrderPromotionDto.kt
package com.sudo248.sudoo.data.dto.order

data class UpsertOrderPromotionDto(
    val promotionId: String,
    val orderSupplierId: String? = null,
    val totalPrice: Double = 0.0,
    val totalPromotionPrice: Double = 0.0,
    val finalPrice: Double = 0.0,
)
